pkg/registry/core/service/strategy.go

```diff
@@ -251,6 +258,17 @@ func loadBalancerPortsInUse(svc *api.Service) bool {
 	return false
 }
 
+func serviceInternalTrafficPolicyInUse(svc *api.Service) bool {
+	if svc == nil {
+		return false
+	}
+	switch svc.Spec.InternalTrafficPolicy {
+	case api.ServiceInternalTrafficPolicyCluster, api.ServiceInternalTrafficPolicyPreferLocal, api.ServiceInternalTrafficPolicyLocal:
```

Comment:
I don't think we need to actually check the values, this can probably just be:
```
if svc == nil {
    return false
}

return svc.Spec.InternalTrafficPolicy != nil 
```
(after we updated InternalTrafficPolicy to string pointer)